test/v2-annotations.json

```diff
@@ -133,10 +133,28 @@
           ],
           "RequiresFilter": true
         }
+      },
+      "GetOne": {
+        "$Function": "V2.Annotations.GetOne"
       }
     },
+    "GetOne": [
```

Comment:
Is one `<FunctionImport>` converted to two properties with the same name, one inside the Container and one outside?